test/check-machines-consoles

```diff
@@ -723,6 +723,7 @@ fullscreen=0
         # cleanup expects self.browser to be logged in.
         self.login_and_go("/machines")
 
+    @testlib.skipWsContainer("Something broken with WebSockets....")
```

Comment:
That also stops the already existing and working test. Please move that into an `if m.ws_container:` condition for the part that doesn't work. FTR, I don't see an immediate reason why it can't work, websocket is all between browser and cockpit-ws. That it runs in a container and is connected to a bridge over ssh shouldn't matter.